### ISSUE TITLE
Expose FsCheck Check.Verbose or introduce Every and EveryShrink in FsCheckConfig

### ISSUE THREAD

**Issue by da9l** [haf/expecto#329]
When learning FsCheck I've found it useful to be able to run a property with Check.Verbose as opposed to Check.One. 

I see that FsCheckConfig is mapped to a FsCheck.Config but the Every and EveryShrink functions (that are used in Check.Verbose) are set to fun _ _ -> String.Empty.

Is there a better way to accomplish this that I've missed? Or would this be the way forward to implement support for Check.Verbose?

**Comment by AnthonyLloyd** [comment]
Yes it looks like Every and EveryShrink could be based on a property on FsCheckConfig. Or maybe it could be based on the Verbosity of the ExpectoConfig.

**Comment by da9l** [comment]
Yeah, the Every and EveryShrink are pretty generic and the Check.Verbose sets a specific implementation. In the Expecto-FsCheck integration would it be sufficient if the ExpectoConfig verbosity setting results in a specific implementation too?

**Comment by AnthonyLloyd** [comment]
I would think so.
